handlers/main.yml

```diff
@@ -4,21 +4,30 @@
     daemon_reload: true
 
 - name: restart peering-manager
-  ansible.builtin.service:
+  become: yes
```

Comment:
Please follow current code style and use `true` or `false`  instead of `yes` or `no` for boolean value. Also put `become` statements after the actual Ansible module code block.